packages/database/zod/insights.ts

```diff
@@ -1,30 +1,33 @@
+import { type Insight, InsightCategory } from "@prisma/client";
 import { z } from "zod";
-import { ZId } from "./common";
+import { ZId } from "../../types/common";
 
-export const ZInsightCategory = z.enum(["featureRequest", "complaint", "praise", "other"]);
+export const ZInsightCategory = z.nativeEnum(InsightCategory);
 
-export type TInsightCategory = z.infer<typeof ZInsightCategory>;
-
-export const ZInsight = z.object({
+const InsightModel = z.object({
   id: ZId,
   createdAt: z.date(),
   updatedAt: z.date(),
   environmentId: ZId,
   title: z.string(),
   description: z.string(),
   category: ZInsightCategory,
```

Comment:
inconsistent naming with `ZInsightCategory` and `InsightModel`. since we now don't distinguish between the typescript types and zod types it might make sense to remove the `Z` prefix, but then we need to do this consistently.